Code/Tools/ProjectManager/Platform/Linux/ProjectUtils_linux.cpp

```diff
@@ -39,16 +37,13 @@ namespace O3DE::ProjectManager
             }
 
             // Look for the first compatible version of clang. The list below will contain the known clang compilers that have been tested for O3DE.
-            for (const QString& supportClangCommand : SupportedClangCommands)
+            for (const QString& supportClangVersion : SupportedClangVersions)
             {
-                auto clangCompilers = supportClangCommand.split('|');
-                AZ_Assert(clangCompilers.length()==2, "Invalid clang compiler pair specification");
-
-                auto whichClangResult = ProjectUtils::ExecuteCommandResult("which", QStringList{clangCompilers[0]}, QProcessEnvironment::systemEnvironment());
-                auto whichClangPPResult = ProjectUtils::ExecuteCommandResult("which", QStringList{clangCompilers[1]}, QProcessEnvironment::systemEnvironment());
+                auto whichClangResult = ProjectUtils::ExecuteCommandResult("which", QStringList{QString("clang-%1").arg(supportClangVersion)}, QProcessEnvironment::systemEnvironment());
+                auto whichClangPPResult = ProjectUtils::ExecuteCommandResult("which", QStringList{QString("clang++-%1").arg(supportClangVersion)}, QProcessEnvironment::systemEnvironment());
                 if (whichClangResult.IsSuccess() && whichClangPPResult.IsSuccess())
                 {
-                    return AZ::Success(supportClangCommand);
+                    return AZ::Success(QString("clang-%1").arg(supportClangVersion));
```

Comment:
You can remove this function now, that it ProjectManager isn't trying to do compiler detection.

Reply:
I couldn't, its a common function that is implemented for all platforms. Project Manager uses it to make sure you have a compatible compiler before you even attempt to generate a project. 

The difference now is that the detected compiler isnt used for the cmake command line anymore.

Reply:
Alright